packages/manager/src/views/Settings/components/Environment/Environment.tsx

```diff
@@ -0,0 +1,221 @@
+                  <Td colSpan={6}>
+                    <EmptyState>
+                      <EmptyStateIcon icon={CubesIcon} />
+                      <Title headingLevel="h4" size="lg">
+                        Environments not found
+                      </Title>
+                      <EmptyStateBody>Please create an environment</EmptyStateBody>
+                    </EmptyState>
+                  </Td>
+                </Tr>
+              )}
+              {envList.isSuccess &&
+                envList.data?.map((env: EnvItem) => (
+                  <Tr key={env._id}>
+                    <Td dataLabel={env.env}>{env.env}</Td>
+                    <Td dataLabel={env.createdAt}>
+                      <Text component={TextVariants.small}>
+                        {new Date(env.createdAt).toUTCString()}
+                      </Text>
+                    </Td>
+                    <Td>
+                      <a href={`https://${env.url}`} target="_blank" rel="noopener noreferrer">
+                        <ExternalLinkAltIcon />
+                        {`${env.url.slice(0, URL_LENGTH_LIMIT)} ${
+                          env.url.length > URL_LENGTH_LIMIT ? '...' : ''
+                        }`}
+                      </a>
+                    </Td>
+                    <Td>
+                      <ClipboardCopy
+                        hoverTip="Copy"
+                        clickTip="Copied"
+                        variant="inline-compact"
+                        isCode
+                      >
+                        {`${window.location.origin}/api/v1/applications/deploy/${env?.propertyIdentifier}/${env?.env}`}
+                      </ClipboardCopy>
+                    </Td>
+                    <Td dataLabel={env.updatedAt}>
+                      <Text component={TextVariants.small}>
+                        {new Date(env.updatedAt).toUTCString()}
+                      </Text>
+                    </Td>
+                  </Tr>
+                ))}
+            </Tbody>
+          </TableComposable>
+        </CardBody>
+      </Card>
+      <Modal
+        title="Create Environment"
+        variant={ModalVariant.medium}
+        isOpen={popUp.createEnv.isOpen}
+        onClose={() => handlePopUpClose('createEnv')}
+      >
+        <CreateEnvForm onClose={() => handlePopUpClose('createEnv')} onSubmit={handleCreateEnv} />
+      </Modal>
+
+      <Modal
+        title={
+          (
+            <Flex alignItems={{ default: 'alignItemsCenter' }}>
+              <Tooltip
+                content={
+                  <div>
+                    SPAship sync is a process in which a schedular pulls HTML pages (from a remote
+                    location) into a specific directory of an environment periodically,
+                    <a
+                      target="_blank"
+                      href="https://source.redhat.com/groups/public/dxp/exd_digital_experience_platforms_dxp_blog/introducing_sync_service_in_spaship"
```

Comment:
We can read it from the Env Variables, later on it might raise an issue as it's an Internal Article Link

Reply:
Fixed